### PR TITLE
[4.0 ] update helioviewer page and run more commands

### PR DESCRIPTION
**Pull request by nabobalis**
See https://helionauts.org/t/sunpy-help-with-querying-helioviewer-org/991/3

### CODE REVIEW

File: changelog/6475.doc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixed bug in un-run code in :ref:`docs/guide/acquiring_data/helioviewer.rst`
```
Comment (nabobalis) [inline changelog/6475.doc.rst:1]: Dont think this is correct.

**Comment by nabobalis** [comment]
online failure on tour.rst file, not from this PR.

**Comment by dstansby** [comment]
Looks good - would probably be better to target `main` and backport?

**Comment by dstansby** [comment]
Just saw the other PR so targeting 4.0 is fine here

**Review by dstansby — APPROVED** [review]

**Pull request merged**